### ISSUE TITLE
[Il ne faut pas vendre la peau de YARN parce qu'un Mesos vaut mieux que deux Kubernetes] Pascal Gillet

### ISSUE THREAD

**Issue by pgillet** [GDGToulouse/meetup-cfp#77]
{ Please fill in this template to help us have a better understanding about you and your talk. Use this pattern for the issue title: [talk title] Firstname Lastname }

Notice: We cannot reimburse travel and hotel expenses unfortunately, we are a non profit association.

Language of the talk: 
- [X] 🇫🇷
- [ ] 🇬🇧

Please use the language you will use during the presentation to fill in this form.

## About you
Je suis ingénieur logiciel, spécialisé dans le Big Data, avec un intérêt croissant pou l'IA/ML/DL.
Je m'intéresse à la gestion des données en général, notamment la valorisation des données ouvertes, les problématiques liées à la capture, le stockage, la recherche, le partage, l'analyse et la visualisation de données massives, et enfin donner du sens aux données avec les standards du Web sémantique.

J'ai rejoint Stack Labs en 2018, une petite structure toulousaine, spécialisée dans le Cloud et le Big Data. Nous sommes tous des passionnés, sans exception.
Stack Labs propose à ses clients des profils essentiellement techniques, et entend redonner ses lettres de noblesse aux développeurs. Stack Labs a en effet compris qu'écrire du code était une activité intellectuelle, parfois complexe, et qui pouvait produire de la valeur.

En dehors du travail, j'aime la verticalité (traduction: je pratique l'escalade, beaucoup).

Site Web: http://pascalgillet31.free.fr/

* Name: Pascal GILLET
* Link to profile picture (best quality you have):
* Job title: Cloud & Big Data Architect
* Company: Stack Labs
* Twitter handle: @pagillet
* Experience as a speaker:
    - [X] beginner
    - [ ] advanced
    - [ ] expert
* Links to your previous events, videos etc. : 
J'ai donné ce même talk au [Snowcamp 2019](https://snowcamp2019.sched.com/event/J2sZ/il-ne-faut-pas-vendre-la-peau-de-yarn-parce-quun-mesos-vaut-mieux-que-deux-kubernetes?iframe=no&w=100%&sidebar=no&bg=no)
---

## Description of your talk (200 characters at least)
Encore une présentation de Kubernetes !? Oui mais attendez...
Maintenant que la hype de K8s est (un peu) redescendue, il est l'heure de prendre un peu de recul et de placer le désormais célèbre framework de Google face à un écosystème bien en place avec notamment Mesos et YARN.

Bien que ces technologies permettent d'utiliser des conteneurs pour déployer, gérer et mettre à l'échelle des applications dans un cluster, elles résolvent en réalité des problèmes bien différents et sont ancrées dans des contextes très divers. En fait, aucun de ces outils largement adoptés n'est complètement comme les autres.
Plutôt que de (simplement) comparer les caractéristiques et les fonctionnalités qui se chevauchent de ces technologies en évolution rapide, revoyons la mission initiale de chaque projet, son architecture et la manière dont ces solutions peuvent se compléter et interagir les unes avec les autres.
On parlera d'ordonnancement et d'orchestration de conteneurs (avec Docker bien évidemment), d'allocation dynamique de ressources, et d'autres traits caractéristiques des gestionnaires de clusters. On évoquera également des schedulers moins connus comme Nomad d'Hashicorp. Et nous nous risquerons à imaginer, pourquoi pas, notre solution idéale: hautement disponible, distribuée, capable de prendre en charge des services conteneurisés, virtualisés, legacy, de courte et longue durée...
Ce Talk aborde des concepts d'informatique distribuée complexes. Pour cette raison, le niveau de ce Talk est fixé à "Intermédiaire".
Il est aussi l'occasion de tenir un discours "engagé" et de nous rappeler à nous-mêmes, acteurs du monde numérique, consultants et salariés des ESN, que nous avons un devoir de conseil et que nous devons accompagner nos clients dans leurs choix techniques. Il faut pour cela nous informer sans cesse par de la veille technologique, comparer des solutions et en dresser un état de l'art, ne pas (tout de suite) céder à la hype des dernières technologies et développer un esprit critique.
À mon sens, il s'agit de rappeler ici la mission principale de l'architecte: être capable d'évaluer objectivement une ou plusieurs solutions selon des critères de coûts, fonctionnels et techniques.

---

## Miscs

* Tags (e.g: angular, android, web, iot) : Architecture, DevOps, Cloud, Container
* Talk level :
    - [] beginner
    - [X] advanced
    - [ ] expert
* Length (max 45 min) : 45 mn
* Format:
    - [X] Slides
    - [ ] Live coding
    - [ ] Video
    - [ ] Sound
* Availability (e.g: ASAP, before/after XXX): After 15/03
[1548839429392_YARN-Mesos-Kubernetes.pdf](https://github.com/GDGToulouse/meetup-cfp/files/2817927/1548839429392_YARN-Mesos-Kubernetes.pdf)

**Comment by shprink** [comment]
Salut, merci pour la proposition! 
Tu parles de Yarn mais c'est pas le gestionnaire de package node si?

**Comment by pgillet** [comment]
Non, YARN pour Yet Another Resource Negotiator, le cluster manager de la plate-forme Hadoop.

**Comment by shprink** [comment]
Ok c'est pour ca que j'avais pas compris le titre de ton talk au Snowcamp  😄
